python/cugraph/dask/opg_pagerank/pagerank.py

```diff
@@ -116,17 +116,17 @@ def pagerank(input_graph,
 
     result = dict([(data.worker_info[wf[0]]["rank"],
                     client.submit(
-            call_pagerank,
-            Comms.get_session_id(),
-            wf[1],
-            data.local_data,
-            alpha,
-            max_iter,
-            tol,
-            personalization,
-            nstart,
-            workers=[wf[0]]))
-            for idx, wf in enumerate(data.worker_to_parts.items())])
+                    call_pagerank,
+                    Comms.get_session_id(),
+                    wf[1],
+                    data.local_data,
+                    alpha,
+                    max_iter,
+                    tol,
+                    personalization,
+                    nstart,
+                    workers=[wf[0]]))
+                   for idx, wf in enumerate(data.worker_to_parts.items())])
```

Comment:
python/cugraph/dask/opg_pagerank/pagerank.py

We're asked to avoid using "opg".

Reply:
Should this be done in this PR for the `pagerank` code or should this be updated in an unifying PR?

Reply:
Anyone feel free to self assign https://github.com/rapidsai/cugraph/issues/996 this needs to complete this week

Reply:
I updated all the code base in my PPR PR. Make sure to sync with it and avoid adding any opg occurrence.